schemas/jsonschema/LineMarker.json

```diff
@@ -0,0 +1,240 @@
+{
+  "$comment": "Generated from LineMarker by @foxglove/message-schemas",
+  "title": "LineMarker",
+  "description": "A marker representing a series of points connected by lines",
```

Comment:
```suggestion
  "description": "A marker representing a series of connected points",
```

Reply:
Took the other suggestions, but I don't like this one as much